src/pages/metapackages/open-source/index.md

```diff
@@ -50,7 +50,15 @@ At this point, you should see symlinks for all the `pwa` modules inside the `ven
 -  Run a Magento installation with additional modules.
 -  Develop locally using the standard git workflow.
 
-You may need to ensure that there are no `Magento_PWA*` modules listed as `enabled` when you run `bin/magento module:status`. If there are, [follow the docs on how to enable modules](https://developer.adobe.com/commerce/php/development/build/component-management/).
+You may need to ensure that there are no `Magento_*Pwa` modules listed as `enabled` when you run `bin/magento module:status`. If there are, [follow the docs on how to enable modules](https://developer.adobe.com/commerce/php/development/build/component-management/).
+
+2. Enable the PWA modules
```

Comment:
```suggestion
1. Enable the PWA modules
```

Reply:
All ordered lists should use `1.` in .md. They'll render in order during the build.